StylableSwiftUI/Classes/StylistIdentifier.swift

```diff
@@ -61,7 +75,13 @@ extension StylistIdentifier: LosslessStringConvertible {
             return self.token
```

Comment:
Do we need to maybe include the theme in this return path?

Reply:
Yup, done